ASUWebPlatforms/webspark-ci
# WS2-1985: allow buttons within divs etc

This will allow buttons to be placed within `<div>` and `<p>` tags.

To test: 
- Navigate to https://pr-375-webspark-ci.ws.asu.edu/buttons
- Click on "Layout" and click the pencil icon and "Configure" for the button area at the top
- Click on "Source" button in the toolbar
- You should see the buttons nested within `<div>` and `<p>` tags.
- Toggle "Source" back and forth. Nothing should be lost.
